Allow any Voting Member to initiate nomination process for adding new role members (#436)
astropy/astropy-project: policies/adding-new-role-members.md

The process is:

> 1. A voting member sends a message to the person nominated, making sure they

Reviewer: ```suggestion
1. A voting member sends a message to the person nominated (with the CoCo in the cc), making sure they
```

Reviewer 2: including @kelle's suggestion (which I agree with), I'd say
```suggestion
1. A voting member or maintainer in a related role sends a message to the person nominated (with the CoCo in the cc), making sure they
```

> 1. A voting member or maintainer in a related role sends a message to the person nominated (with the CoCo in the cc), making sure they
>    understand the role, its responsibilities, and confirming that the nominee agrees. 

Author: ```suggestion
1. A voting member or maintainer in a related role sends a message to the person nominated
   (with the Coordination Committee in the cc), making sure they understand the role, 
   its responsibilities, and confirming that the nominee agrees. 
```
should spell out here as below since this is the first occurrence
